### DISCUSSION TITLE
Prepare for SkiaSharp 3.0.0

### DISCUSSION THREAD

**Discussion by taublast** [Mapsui/Mapsui#2763] [Ideas]
Hi, do you plan on preparing for SkiaSharp 3.0 in advance by starting to support 3.0.0-preview.4.1? 
In theory the stable 3.0 is "for the end of the year" so..

**Comment by inforithmics** [comment]
This is the pull request https://github.com/Mapsui/Mapsui/pull/2603, when Skiasharp gets stable than this will be merged. 
Most incompatibilites have been fixed in the latest mapsui preview so simple using mapsui Preview with nightly skiasharp should work.
see here https://github.com/mono/SkiaSharp/issues/2999
nightly skiasharp
https://pkgs.dev.azure.com/xamarin/public/_packaging/SkiaSharp/nuget/v3/index.json

**Reply by inforithmics** [reply]
Version 3.0.0-preview.5.4 of SkiaSharp is released so everything should work now. Waiting for Release
